docker/cronjob.yaml

```diff
@@ -0,0 +1,50 @@
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  name: k8s-pod
+  namespace: prod
+---
+apiVersion: batch/v1beta1
+kind: CronJob
+metadata:
+  name: jira-reporter
+spec:
+  schedule: "* * * * *"
+  successfulJobsHistoryLimit: 1
+  failedJobsHistoryLimit: 1
+  jobTemplate:
+    spec:
+      template:
+        spec:
+          serviceAccountName: k8s-pod
+          initContainers:
+          - args:
+            - secret/chef/jira/jira-reporter
+            env:
+            - name: VAULT_ADDR
+              value: active.vault.service.sjc.consul:8200
+            - name: ENV
+              value: prod
+            name: secrets
+            image: artifactory.wikia-inc.com/ops/init-vault:0.0.37
```

Comment:
You can bump the version here also just to keep it the latest.

Reply:
Done